node/src/stream_handler_pool.rs

```diff
@@ -533,18 +550,19 @@ impl StreamHandlerPool {
             peer_addr,
             msg.context.data.len()
         );
-        let recipient = self
+        let schedule_message_sub = self
```

Comment:
I think we need to call these variables rather `scheduled_message_sub` as you hardly can have a recipient for a wide range of messages (even if as a generic argument in the MessageScheduler).